loops.js

```diff
@@ -46,7 +60,15 @@
 // ==========================================
 
 
-
+for (let i = 0; i <= 9; i++) {
+    if (i < 3 ) {
+        console.log(i);
+    } else if (i >= 3 && i < 6) {
```

Comment:
```suggestion
    } else if (i < 6) {
```
Omdat de eerste if false was, moet het getal wel groter of gelijk zijn aan 3